Add docs for the UniFi Protect button platform (#20967)
home-assistant/home-assistant.github.io: source/_integrations/unifiprotect.markdown

* **Camera** - A camera entity for each camera channel and RTSP(S) combination found for each camera (up to 6). Only the highest resolution RTSPS camera entity will be enabled by default.
* **Media Player** - If your camera has a speaker, you will get a media player entity that allows you to play audio to your camera's speaker. Any audio file URI that is playable by FFmpeg will be able to be played to your speaker, including via the [TTS Say Service](https://www.home-assistant.io/integrations/tts/#service-say).

> * **Button** - A disabled by default button entity is added for each camera entity. The button will let you reboot your camera device.

Reviewer: Do cameras get a button?
Please describe the functionality in the list, not the entity platforms.

### Floodlight

Each UniFi Protect flood light will get the following entities added:

> * **Button** - A disabled by default button entity is added for each floodlight entity. The button will let you reboot your floodlight device.

Reviewer: Same as above.

> * **Button** - A button entity that is disabled by default will be added for each flood light entity. This will let you Reboot your flood light device.

Reviewer 2: ```suggestion
* **Button** - A disabled by default button entity is added for each floodlight entity. The button will let you Reboot your floodlight device.
```

### Sensor

Each UniFi Protect sensor will get the following entities added:

> * **Button** - A disabled by default button entity is added for each sensor entity. The button will let you reboot your sensor device.

Reviewer: Same as above. I find this rather confusing actually... is it a button or a sensor?

> * **Button** - A button entity that is disabled by default will be added for each sensor entity. This will let you Reboot your sensor device.

Reviewer 2: ```suggestion
* **Button** - A disabled by default button entity is added for each sensor entity. The button will let you Reboot your sensor device.
```

### Viewport

Each UniFi Protect viewport will get the following entities added:

> * **Button** - A disabled by default button entity is added for each viewport entity. The button will let you reboot your viewport device.

Reviewer: Same as above

> * **Button** - A button entity that is disabled by default will be added for each viewport entity. This will let you Reboot your viewport device.

Reviewer 2: ```suggestion
* **Button** - A disabled by default button entity is added for each viewport entity. The button will let you Reboot your viewport device.
```
